acs-integration-tests/src/test/java/com/ge/predix/acceptance/test/policy/admin/PolicyCreationStepsDefinitions.java

```diff
@@ -46,6 +46,7 @@
  *
  * @author 212338046
  */
+//CHECKSTYLE:OFF
```

Comment:
we should probably add notes everywhere "CHECKSTYLE:OFF" and "CHECKSTYLE:ON" are used mentioning why we're doing so